server/routes/users.js

```diff
@@ -0,0 +1,46 @@
+const bcrypt = require('bcrypt');
+const excuteQuery = require('../db/db');
+const uuid = require('uuid');
+const validator = require('validator');
+
+const signup = async (request, reply) => {
+   const { user_mail, user_password } = request.body;
```

Comment:
Our cross team agreement is to use camelCase for variables. I understand that you wanted to make it consistent with DB naming, which is case insensitive. However in code you still need to keep using camelCase as agreed. And only when you are communicating to database you can use snakecase. 
Currently you leak this DB specific detail not only in server code but also to client code, because they will need to send body object with these snake_cased properties.

Reply:
OK, I've corrected it.